src/main/java/com/rtb/projectmanagementtool/project/UserProjectRole.java

```diff
@@ -0,0 +1,14 @@
+// UserProjectRole.java - enum representing role of user in a project
+//
+// REGULAR - regular user with limited capabilities within project.
+// ADMIN - user with more administrative capabilities within project
+// CREATER - the creator of the project. Does everything an admin can do
+// and more, including being able to add/remove admins and delete the project
+
+package com.rtb.projectmanagementtool.project;
+
+public enum UserProjectRole {
+  CREATOR,
+  ADMIN,
+  REGULAR;
```

Comment:
nit:  I wonder if there's another term we can use besides "REGULAR"?  It's hard to reason about how regular is different from the other types, since you can also argue that the ADMIN could be a "regular admin", if that makes sense?

Couple options, maybe project "MEMBER", "PARTICIPANT", "SUBSCRIBER" (although we're not using a subscription design pattern here), "CONTRIBUTOR"